src/jit/compiler.hpp

```diff
@@ -105,7 +105,14 @@ template<typename T>
 inline
 T    genFindLowestBit(T value)
 {
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable:4146) // unary minus operator applied to unsigned type, result still unsigned
+#endif
     return (value & -value);
```

Comment:
I believe, the least ugly way to suppress this warning is (value & (0 - value)).  I haven't tried it in this case, but if it works I would use that.